.github/workflows/release-and-publish.yml

```diff
@@ -0,0 +1,55 @@
+# This workflow opens and updates a pull request with a new package version
+# based on code changes. Merging the pull request updates the version in
+# composer.json, updates the changelog and creates release tags.
+
+# This workflow also publishes the package into the Packagist repository after
+# a new version has been released.
+
+# For more information, see https://github.com/marketplace/actions/release-please-action
+
+
+# PHP Packagist works slightly differently then the other two cases (npm and rubygems). 
+# In this case, "publishing a # package" means calling a Packagist webhook which causes Packagist to scan the GitHub repo 
```

Comment:
@prokopsimek?
```suggestion
# In this case, "publishing a package" means calling a Packagist webhook which causes Packagist to scan the GitHub repo 
```

Reply:
@Toreno96 I'm sorry, I missed it. I'm tired today. I apologize for such a fundamental mistake.

Reply:
Don't worry, it can happen to anyone :)